metadata.yaml

```diff
@@ -28,4 +28,4 @@ resources:
   juju-autoscaler-image:
     type: oci-image
     description: OCI Image for kubernetes-autoscaler with a juju cloud provider.
-    upstream-source: rocks.canonical.com/cdk/cluster-autoscaler-juju:dev-2.9.22-build4
+    upstream-source: rocks.canonical.com/cdk/cluster-autoscaler-juju:dev-2.9.22-build??
```

Comment:
DO NOT MERGE without a proper `upstream-source`

Reply:
New image is now available [here](https://rocks.canonical.com/v2/cdk/cluster-autoscaler-juju/tags/list). The new tag is dev-2.9.latest-build0 as the juju api is using the tip of the main branch that is not part of a juju release yet.